examples/llava/clip.cpp

```diff
@@ -1676,6 +1676,10 @@ void clip_free(clip_ctx * ctx) {
     ggml_free(ctx->ctx_data);
     gguf_free(ctx->ctx_gguf);
 
+    ggml_backend_buffer_free(ctx->params_buffer);
+    ggml_backend_buffer_free(ctx->compute_buffer);
```

Comment:
`compute_buffer` doesn't seem to be used. Can you remove it from `clip_ctx` instead?